knowledge/science/astronomy_tech/radioisotope_termo_generator/qna.yaml.txt

```diff
@@ -0,0 +1,19 @@
+created_by: 
+domain: astronomy_technology
+seed_examples:
+- answer: | Space exploration missions require safe, reliable, long-lived power systems to provide electricity and heat to spacecraft and their science instruments
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!